tests/js/query-tests.js

```diff
@@ -77,18 +75,24 @@ function runQuerySuite(suite) {
 
     for (var index in suite.tests) {
         var test = suite.tests[index];
+        var type;
```

Comment:
Because `var` is function-scoped, not block scoped, and we can't use `let` in this file since it must be ES5.